CameraOverlay/overlay.py

```diff
@@ -134,14 +136,17 @@ def __init__(self, width=1280, height=740):
 		self.data_canvas = Canvas(self.width, self.height)
 		self.message_canvas = Canvas(self.width, self.height)
 
+		configs = read_configs()
+		bike_version = bike or configs["bike"] or DEFAULT_BIKE
```

Comment:
Is it worth writing a unit test just to make sure this works?

Reply:
I did think about that, but I thought things would get complicated when checking that it correctly falls back to an environment variable